lib/jelix/core/defaultconfig.ini.php

```diff
@@ -446,11 +446,13 @@
 [forms]
 ; define input type for datetime widgets : "textboxes" or "menulists"
 controls.datetime.input = "menulists"
+controls.time.input = "menulists"
 ; define the way month labels are displayed widgets: "numbers", "names" or "shortnames"
 controls.datetime.months.labels = "names"
 ; define the default config for datepickers in jforms
 datepicker = default
 datetimepicker = default
+timepicker = default
```

Comment:
set an empty value. We don't have a JS widget to display a timepicker, so no configuration.